internal/clients/embed.go

```diff
@@ -0,0 +1,54 @@
+package clients
+
+import (
+	"context"
+	"embed"
+	"errors"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+var AwsSupported = make(map[string]interface{})
+var ErrArchitectureNotSupported = errors.New("architecture is not supported")
+
+const x86_64 = "x86_64"
+const arm64 = "arm"
```

Comment:
I suggest to use "arm64" which is more appropriate than just "arm" although AWS does not use that.